modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-aws_{context}"]
+
+= Adding an ARM64 compute machine set to your cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.
```

Comment:
This reads a bit confusing to me:
> This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.

I read "This" as "The compute machine set", since machine sets create more machines (i.e., "add compute nodes to your cluster"). But the machine set does not create a cluster, so the second half threw me off. Maybe something like this? Not sure if it's accurate but giving it a try :slightly_smiling_face: 

> The compute machine set adds ARM64 compute nodes to your cluster so that your cluster has multi-architecture compute machines.

Reply:
Ohhhh, yeah I see what you're saying. Yeah the "creates a cluster" part of my original sentence kinda makes that sounds confusing. Thank you!